.bazelrc

```diff
@@ -1,6 +1,9 @@
 ############################################################################
 # All default build options below.
 
+# Required by OpenXLA
+build --nocheck_visibility
```

Comment:
Is there a plan to remove the need for this? Maybe add a TODO with an issue number if one exists?

Reply:
Added a link to the upstream github issue.